lib/redox/patient_admin/discharge.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require "redox"
+
+module Redox
+  module PatientAdmin
+    class Discharge < Models::Message
+      redox_property :Patient, coerce: Models::Patient
+      redox_property :Visit, coerce: Models::Visit
+      redox_property :Observations, coerce: Array[Models::Observation]
```

Comment:
Can you show me where you found the link between PatientAdmin.Discharge and Observations?

I ask because I cannot find a reference to Observations on the [API reference for PatientAdmin.Discharge](https://developer.redoxengine.com/data-models/PatientAdmin.html#Discharge) and I worry I might be missing something.

Reply:
That is a mistake. Thanks!